<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsProjectLayerGroupDialogBase</class>
 <widget class="QDialog" name="QgsProjectLayerGroupDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select layers and groups to embed</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mProjectFileLabel">
       <property name="text">
        <string>Project file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mProjectFileLineEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="mBrowseFileToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QgsLayerTreeView" name="mTreeView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>0</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsLayerTreeView</class>
   <extends>QTreeView</extends>
   <header>qgslayertreeview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mProjectFileLineEdit</tabstop>
  <tabstop>mBrowseFileToolButton</tabstop>
  <tabstop>mTreeView</tabstop>
  <tabstop>mButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
